.github/workflows/ci-build-image.yml

```diff
@@ -129,3 +141,40 @@ jobs:
     - uses: skjolber/maven-cache-github-action@v1
       with:
         step: save
+
+  retest-integration-tests:
+    runs-on: ubuntu-latest
+    if: ${{ !inputs.skip-itests && inputs.run-tests-only && inputs.pr-number != '' && inputs.sha-value != '' }}
+    steps:
+    - name: Install xpath
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y libxml-xpath-perl
+    - name: Install qemu
+      if: ${{ inputs.build-arch != 'amd64' }}
+      continue-on-error: true
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y qemu-user-static
+    - uses: actions/checkout@v4
+      with:
+          repository: ${{ github.repository_owner }}/cryostat
+          ref: ${{ inputs.checkout-ref }}
+          submodules: true
+          fetch-depth: 0
+    - uses: actions/setup-java@v3
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+    - name: Pull cryostat image
+      run: podman pull ghcr.io/${{ github.repository_owner }}/cryostat:pr-${{ inputs.pr-number }}-${{ inputs.sha-value }}-linux-${{ inputs.build-arch }}
+    - name: Run integration tests
```

Comment:
the integration tests are failing right now because it can't access the maven files, such as the -core jar. You'll need to set-up maven with maven-settings-action